nexus/post.ts

```diff
@@ -103,14 +103,20 @@ schema.objectType({
     t.model.images()
     t.model.publishedAt()
     t.int('commentCount', {
-      resolve(parent, _args, ctx, _info) {
-        return ctx.db.comment.count({
+      resolve: async (parent, _args, ctx, _info) => {
+        const threadCommentCount = await ctx.db.comment.count({
```

Comment:
We can do these two count queries concurrently, ala 

```
const [threadCommentCount, postCommentCount] = await Promise.all([<queries here>])
```

Reply:
Ah yes, great suggestion! 🙏🏼 🦔